docs/configuration.md

```diff
@@ -1023,6 +1023,26 @@ Apart from these, the following properties are also available, and may be useful
     The max size of event log file before it's rolled over.
   </td>
 </tr>
+<tr>
+  <td><code>spark.history.fs.eventLog.rolling.maxFilesToRetain</code></td>
+  <td>Int.MaxValue</td>
+  <td>
+    The maximum number of event log files which will be retained as non-compacted. By default,
```

Comment:
Spark currently doesn't have a "release notes" document in the docs dir, but I'm wondering if we should mention somewhere that this is a new feature and may not be completely stable (i.e. use with caution, it may delete more data than you expect, cause some UI issues we haven't thought about, etc). There are migration guides, but that doesn't seem like the right place...

Reply:
I think we are using JIRA issue to write content for release note if it's necessary. Labeled as "release-notes" and content in "Docs text".
https://issues.apache.org/jira/browse/SPARK-25016

I'd be pretty much appreciated if you can write the release note content for this issue, as JIRA issue is not good to review and make change continuously.